_together/03-whats-next.md

```diff
@@ -9,6 +9,7 @@ permalink: /together/whats-next/
 slug: whats-next
 hero_title: What’s next
 hero_subtitle: Toward pattern-driven development for the design system.
+excerpt: ""
```

Comment:
Removes excerpts that we don't use anyway for this page, since they were throwing Jekyll warnings